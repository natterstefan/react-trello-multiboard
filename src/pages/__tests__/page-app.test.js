import React from 'react'
import { shallow } from 'enzyme'

import AppPage from '../page-app'

describe('Page/AppPage', () => {
  test('should render without throwing an error', () => {
    const wrapper = shallow(<AppPage />)
    expect(wrapper).toMatchSnapshot()
  })
})
