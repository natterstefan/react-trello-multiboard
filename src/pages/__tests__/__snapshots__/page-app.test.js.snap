// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page/AppPage should render without throwing an error 1`] = `<Connect(WithStyles(MainApp)) />`;
